### PR TITLE
Hype Group layout object

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

This is (probably) the last major new pattern for the new style of marketing landing page, the Hype Group. It arranges intro, content and visual object blocks in a stylized manner well suited for this sort of content.

Also makes a small change to Cloud Cover to constrain the z-indices of its clouds.

### Narrow screens

Screen Shot 2023-05-22 at 09 24 49

### Wide screens

Screenshot 2023-05-22 at 09-21-03 objects-hype-group--multiple

### Docs

Screenshot 2023-05-22 at 09-22-12 objects-hype-group--multiple

## Testing

Try out [the deploy preview](https://deploy-preview-2170--cloudfour-patterns.netlify.app/?path=/story/objects-hype-group--multiple) in the following browsers:
- [x] Chrome
- [x] Edge
- [x] Firefox
- [x] Safari

### CODE REVIEW

File: src/objects/hype-group/hype-group.scss
Comment (spaceninja) [inline src/objects/hype-group/hype-group.scss:1]: The comments on this file are great, thank you!

**Comment by spaceninja** [comment]
Screenshot 2023-05-22 at 3 25 51 PM

The last item gets cut off on the bottom. I'm not sure if that's just an artifact of how it's rendered in Storybook or if it would happen in a real context, though.

**Review by spaceninja — APPROVED** [review]
Code looks great and it works as expected in browsers, with one possible bug that I left in a comment.

I'm approving on the assumption that it's not a real bug, just a Storybook artifact, but if it turns out to be real, I'm happy to re-review after you apply a fix.

**Comment by tylersticka** [comment]
> The last item gets cut off on the bottom. I'm not sure if that's just an artifact of how it's rendered in Storybook or if it would happen in a real context, though.

I believe this to be an artifact of Storybook, since these items intentionally overflow.

**Pull request merged**